### DISCUSSION TITLE
Sorting Classes with Precommit Hook

### DISCUSSION THREAD

**Discussion by jaobrown** [tailwindlabs/tailwindcss#1812] [Help]
Hello!

I have been using [Headwind](https://github.com/heybourn/headwind) in vscode to sort tailwind classes as I author html with tailwind. I can't seem to find any solutions for performing these sorting functions via a precommit hook, using something like [Husky](https://github.com/typicode/husky). 

Has anyone gotten this to work? I'm not worried about the order, as I don't have a massive preference—I'm just hoping to automate consistency in the markup. Working with a few other devs at the moment, and its difficult to keep the ordering consistent. 

Any advice would be appreciated 👍

**Comment by mengxi-ream** [comment]
use official plugin with prettier, and combine prettier with husky
https://github.com/tailwindlabs/prettier-plugin-tailwindcss
